conda/cli/activate.py

```diff
@@ -3,14 +3,25 @@
 import os
 import sys
 from os.path import isdir, join, abspath
+import errno
 
 from conda.cli.common import find_prefix_name
+import conda.config
```

Comment:
Moving these imports seems to be what is causing the horrible logging messages.  I have moved them back below.

This really needs a real fix.  Local imports to fix errors are a sign of a deeper problem.